chia/cmds/wallet.py

```diff
@@ -99,18 +99,19 @@ def show_cmd(wallet_rpc_port: int, fingerprint: int) -> None:
     asyncio.run(execute_with_wallet(wallet_rpc_port, fingerprint, {}, print_balances))
 
 
-@wallet_cmd.command("get_address", short_help="Get a wallet receive address")
+@wallet_cmd.command("get_address", short_help="Get the current public wallet address")
 @click.option(
     "-wp",
     "--wallet-rpc-port",
     help="Set the port where the Wallet is hosting the RPC interface. See the rpc_port under wallet in config.yaml",
     type=int,
     default=None,
 )
+@click.option( "-n", "--new_address", help="Generate new public wallet address.", type=bool, default=False, is_flag=True, show_default=True)
```

Comment:
I like being able to explicitly specify both directions.

https://click.palletsprojects.com/en/7.x/options/#boolean-flags

Though, I can't swear to the `"-n/-o"` bit.  Probably ought to check that.  Maybe it should be new/old, maybe new/current, new/most recent, existing?  I"m not quite sure.

```suggestion
@click.option("-n/-o", "--new_address/--old_address", help="Generate new public wallet address.", default=False, show_default=True)
```

Reply:
As far as i can see in the wallet_rpc_api.py is only the option to generate a new address with an random puzzle hash. To get the previous address again, we have save the current/old puzzle_hash somewhere. But i dont know how the best way would.
Line 693 generates the new address in the backend:
https://github.com/Chia-Network/chia-blockchain/blob/0096f755a1d1fcacba621fa35c67d111e81be2d0/chia/rpc/wallet_rpc_api.py

Reply:
Sorry I wasn't more clear.  I meant that I would like to be able to explicitly request the existing behavior by passing an option rather than only by not passing an option.  I don't expect to be able to go backwards through the list.  So, something like `--current_address` would provide the existing behavior while `--new_address` would provide the new behavior.  I'll let you know if I figure out what name I prefer...